internal/common/baseclient/baseclient.go

```diff
@@ -3,6 +3,7 @@ package baseclient
 
 import (
 	"fmt"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
```

Comment:
this should be moved below automatically by `gofmt` in `vscode`--URL imports should be below of intern libs.